server/src/main/java/io/spine/server/stand/UpdateHandler.java

```diff
@@ -87,13 +77,14 @@ void handle(EventEnvelope event) {
      * the information on the passed event.
      */
     private void checkActive(EventEnvelope event) {
-        checkState(
-                isActive(),
-                "Dispatched an event of type `%s` to the non-active subscription with the ID `%s`.",
-                event.typeUrl(),
-                subscription.getId()
-                            .getValue()
-        );
+        if (!isActive()) {
+            throw newIllegalStateException(
```

Comment:
Why? `checkState` seems more obvious (as it does not have a negation).

Reply:
To avoid calculation of `TypeUrl` all the time.